sdk/assets/data/working_with_mltable.ipynb

```diff
@@ -0,0 +1,290 @@
+{
+ "cells": [
```

Comment:
should we also update https://github.com/Azure/azureml-examples/blob/main/sdk/assets/data/data.ipynb?